app/javascript/vite/application.js

```diff
@@ -24,6 +27,31 @@ import * as ScpApi from '~/lib/scp-api'
 
 window.SCP = window.SCP ? window.SCP : {}
 
+// Initialize Sentry to enable logging JS errors to Sentry
+Sentry.init({
+  dsn: 'https://a713dcf8bbce4a26aa1fe3bf19008d26@o54426.ingest.sentry.io/1424198',
+  integrations: [new BrowserTracing()],
+
+  // send 100% of the transactions to Sentry since they will only occur on errors
+  tracesSampleRate: 1.0
```

Comment:
Out of curiosity, is there a setting that would let us do something like "Don't log more than 500 errors per user per day?"  DSP Sentry clients sometimes overlog, risking disabling Sentry for all other DSP clients until quota refresh, and we often only become aware of such incidents retroactively.

It'd be wonderful if Sentry clients like this had an easy way to cap or throttle their own logging _after_ reaching some client-defined threshold, to avoid such systemic risks for observability.